test/lib/naming-assert.js

```diff
@@ -0,0 +1,24 @@
+var mockAndAssert = require('./mock-and-assert'),
+    opts = { scheme: 'flat' },
+    convention = {
```

Comment:
BTW, shouldn't all the conventions be kept somewhere in `bem-naming` and reused here?